numpy/_core/code_generators/generate_umath.py

```diff
@@ -275,6 +276,10 @@ def english_upper(s):
     'I': 'uint',
     'l': 'long',
     'L': 'ulong',
+    # We sometimes need int64, but we have no obvious char for it, use k and
+    # define it as `int64` below.
+    'k': 'int64',
+    'K': 'uint64',
```

Comment:
It's not clear to me how [this page](https://numpy.org/doc/1.26/reference/arrays.scalars.html) get generated - but maybe it needs to be rearranged now that `int64` isn't a sized alias anymore? And will 'k' show up somewhere on that page?

Reply:
`k` isn't a thing, it is only used in this file internally to define some ufunc types.  That said, you are right that I didn't really change the docs yet, and that page looks like it needs to be expanded to list the new `long`!

Reply:
Ah, it wasn't clear to me it wasn't user-facing. Thanks!